sample_consensus/include/pcl/sample_consensus/sac.h

```diff
@@ -56,10 +55,6 @@ namespace pcl
   {
     typedef typename SampleConsensusModel<T>::Ptr SampleConsensusModelPtr;
 
-    private:
-      /** \brief Constructor for base SAC. */
-      SampleConsensus () {};
```

Comment:
The implication of this change is that there is now a _public_ default constructor. Is this a conscious change on your part? What are the reasons?

Reply:
Oops, that's a wrong statement. Forget it.